src/sidebar/media-embedder.js

```diff
@@ -110,6 +110,7 @@ function youTubeQueryParams(link) {
     't', // will be translated to `start`
   ];
   const linkParams = queryString.parse(link.search);
+  /** @type {Record<string, string>} */
```

Comment:
Just because you were in here, or did something else prompt this change?

Reply:
> Updated: Because of the typing introduced in the new implementation?

Yes, exactly.